tests/test_on_data/ophys/test_imaging_interfaces.py

```diff
@@ -903,3 +905,201 @@ def test_incorrect_folder_structure_raises(self):
+
+    data_interface_cls = InscopixImagingInterface
+    save_directory = OUTPUT_PATH
+    interface_kwargs = dict(
+        file_path=str(OPHYS_DATA_PATH / "imaging_datasets" / "inscopix" / "movie_longer_than_3_min.isxd")
+    )
+    optical_series_name = "OnePhotonSeries"
+    conversion_options = dict(photon_series_type="OnePhotonSeries")
+
+    def check_extracted_metadata(self, metadata: dict):
+        """Test metadata extraction for file with rich acquisition info."""
+
+        # NWBFile checks
+        nwbfile = metadata["NWBFile"]
+        assert nwbfile["session_start_time"] == datetime(2019, 10, 7, 16, 22, 1, 524186)
+        assert nwbfile["session_id"] == "4D_SAAV_PFC_IM7_20191007"
+
+        # Device checks
+        device = metadata["Ophys"]["Device"][0]
+        assert device["name"] == "NVista3"
+        device_desc = device.get("description", "")
+        if device_desc:
+            assert "Inscopix Microscope" in device_desc
+            assert "Serial: FA-11092903" in device_desc
+            assert "Software: 1.3.0" in device_desc
+
+        # ImagingPlane checks
+        imaging_plane = metadata["Ophys"]["ImagingPlane"][0]
+        assert imaging_plane["name"] == "ImagingPlane"
+
+        desc = imaging_plane.get("description", "")
+        assert "The plane or volume being imaged by the microscope." in desc
+        assert "Exposure Time (ms): 20" in desc
+        assert "Gain: 5.9" in desc
+        assert "Focus: 1000" in desc
+        assert "eFocus: 370" in desc
+        assert "EX LED Power (mw/mm^2): 0.4" in desc
+        assert "OG LED Power (mw/mm^2): 0.2" in desc
+
+        # Optical channel checks
+        optical_channel = imaging_plane["optical_channel"][0]
+        assert optical_channel["name"] == "channel_0"
+        assert (
+            optical_channel["description"] == "An optical channel of the microscope."
+        )  # Default metadata because this was not included in the source metadata
+
+        # OnePhotonSeries checks
+        ops = metadata["Ophys"]["OnePhotonSeries"][0]
+        assert ops["name"] == "OnePhotonSeries"
+        assert (
+            ops["description"] == "Imaging data from one-photon excitation microscopy."
+        )  # Default metadata because this was not included in the source metadata
+        assert ops["unit"] == "n.a."  # Default metadata because this was not included in the source metadata
+        assert ops["imaging_plane"] == "ImagingPlane"
+        assert ops["dimension"] == [33, 29]
+
+        # Subject checks
+        subject = metadata["Subject"]
+        assert subject["subject_id"] == "Unknown"
+        assert subject["species"] == "Unknown species"
+        assert subject["sex"] == "M"
+        assert "description" not in subject
+        assert "weight" not in subject
+        assert "date_of_birth" not in subject
+
+
+@skip_on_python_313
+@skip_on_darwin_arm64
+class TestInscopixImagingInterfaceMovieU8(ImagingExtractorInterfaceTestMixin):
```

Comment:
I just realize, this one says in the docstring that is volumetric:

https://github.com/catalystneuro/roiextractors/blob/75afb5b4eedadf22a2a42cc22d8e5e2287a6e192/tests/test_inscopiximagingextractor.py#L111

But when you calculate `get_series` it seems that we are returning data as planar:

https://github.com/catalystneuro/roiextractors/blob/75afb5b4eedadf22a2a42cc22d8e5e2287a6e192/tests/test_inscopiximagingextractor.py#L136-L138

What is going on here?

Reply:
Ok, thanks.